OasisDEX/oasis-borrow
# Ades/gas estimation/account for slippage when decreasing multiple

# [Title](https://app.shortcut.com/oazo-apps/story/5120/bug-gas-estimate-continuously-fails-when-decreasing-multiple)
  
## Changes 👷‍♀️
  <Please add short list of changes>
- accounted for slippage when decreasing multiple
  
## How to test 🧪
- open a position
- decrease multiple.
- ensure gas is estimated
